docs/Getting_Involved/Newcomers.rst

```diff
@@ -0,0 +1,116 @@
+Welcome to the Newcomers guide!
+===============================
+
+This guide will help you to get involved and teach you how to make your first
+contribution to coala!
+
+Meet the community!
+-------------------
+
+To get started, the first way is to meet the community. We use gitter to
+communicate, and there the helpful community will guide you.
+
+::
+
+    https://gitter.im/coala-analyzer/coala
+
+
+Congratulations! You are now part of our community.
```

Comment:
maybe something like the newcomer could ping a hello world to us to actually get in touch?